app/src/main/java/com/example/admin/learningandroid/Activity_of_Single_Instance.java

```diff
@@ -0,0 +1,90 @@
+package com.example.admin.learningandroid;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+public class Activity_of_Single_Instance extends AppCompatActivity {
+    String Tag = "Single_Instance";
```

Comment:
this can be static final and the declaration can be like     public static final int SOME_CONSTANT = 42;
Please read https://source.android.com/setup/contribute/code-style